### PR TITLE
Hashing in C++ #2211

### PR DESCRIPTION
**Pull request by elixir14082002**
<hr>

## Description 📜

Added documentation and a video on Hashing in CPP.
#2211 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by elixir14082002** [comment]
@Suvraneel Please review my code.

**Comment by Suvraneel** [comment]
@Shivani0701 @deeqakk Review the doc... Video is cool :rocket:

File: C_CPP/Data Structures/Hashing/Hashing.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    for(auto i:m)
+    {
+        cout<<i.first<<" "<<i.second<<endl;
+    }
+}
+```
+We now, can clearly understand the implementation of unordered maps and sets.
+
+<h3>Here is an explaination of the code,</h3>
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Hashing/Hashing.md:108]:
```suggestion
<h3>Here is an explanation of the code,</h3>
```

File: C_CPP/Data Structures/Hashing/Hashing.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+<hr>
+https://drive.google.com/file/d/1AT8uLDtcFEc1wnQ4lkFKuAdx3cDUT4lI/view?usp=sharing
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Hashing/Hashing.md:110]:
```suggestion
[Suitable Title by Elixir](https://drive.google.com/file/d/1AT8uLDtcFEc1wnQ4lkFKuAdx3cDUT4lI/view?usp=sharing)
```

**Review by Suvraneel — CHANGES_REQUESTED** [review]

**Comment by elixir14082002** [comment]
@Suvraneel I have made the changes. Please review!

**Comment by Suvraneel** [comment]
@ChetasShree @gunjan-g

**Review by AnirbanBall — APPROVED** [review]
Looking good to me ✌️ .Can be merged .

**Comment by Shivani0701** [comment]
> @Shivani0701 @deeqakkk Review the doc... Video is cool 🚀

@Suvraneel It is perfect ✌️

**Comment by gunjan-g** [comment]
> @ChetasShree @gunjan-g Doc dekh lo ekbar

Yes, It's good

**Review by deeqakkk — APPROVED** [review]

**Comment by Suvraneel** [comment]
@elixir14082002  This is nice ... Just 1 last thing...
In the readme for Data Structure add a relative link to this

**Comment by elixir14082002** [comment]
@Suvraneel I have updated the readme for Data Structures.

**Comment by Suvraneel** [comment]
@elixir14082002 Awesome ❤️  Make more video contribs :octocat: 
Also, just 1 suggestion, instead of bits std, from next one show them the actual headers (Knowledge is essential)

**Pull request merged**